zds/member/views.py

```diff
@@ -456,18 +454,18 @@ def unregister(request):
             vote.comment.dislike -= 1
         vote.comment.save()
     votes.delete()
-    # all messages anonymisation (forum, article and tutorial posts)
+    # All contents anonymization.
     Comment.objects.filter(author=current).update(author=anonymous)
     PrivatePost.objects.filter(author=current).update(author=anonymous)
     CommentEdit.objects.filter(editor=current).update(editor=anonymous)
     CommentEdit.objects.filter(deleted_by=current).update(deleted_by=anonymous)
-    # karma notes, alerts and sanctions anonymisation (to keep them)
+    # Karma notes, alerts and sanctions anonymization (to keep them).
     KarmaNote.objects.filter(moderator=current).update(moderator=anonymous)
     Ban.objects.filter(moderator=current).update(moderator=anonymous)
     Alert.objects.filter(author=current).update(author=anonymous)
     Alert.objects.filter(moderator=current).update(moderator=anonymous)
     BannedEmailProvider.objects.filter(moderator=current).update(moderator=anonymous)
-    # in case current has been moderator in his old day
+    # In case current has been moderator in the past.
```

Comment:
Ah. Elle a été définie plus de 20 lignes plus haut, cette variable. Du coup ce commentaire n'est pas compréhensible.

Quelques possibilités : 

*     In case `current` has been moderator
* Le commentaire est à la ligne avant ou après la variable en question
* `current user`